public/stylesheets/main.css

```diff
@@ -9,6 +9,11 @@
     src: url(/fonts/fabrica.otf);
 }
 
+@font-face {
+    font-family: sourcesanspro;
```

Comment:
Can we change this to having some casing or spacing applied? How about dashes for word breaks?